include/node.h

```diff
@@ -124,6 +149,29 @@ class Node {
   double pdd_smoothing_delta_{0.2};
   double pdd_slope_{1e-12};
 
+  //! leak diameter
+  double leak_diameter_{0};
+  //! Value of pi
+  double pi_{3.1415926};
+  //! leak area
+  double leak_area_{0};
+
+  //! accleration due to gravity
+  double g_{9.81};
+  //! leak dischage (static and iter)
+  double leak_discharge_{0};
+  double iter_leak_discharge_{0};
+  //! leak coefficients
+  double leak_discharge_coefficient_{0.75};
+
+  //! polynomial coefficients for demand-pressure leak-pressure
+  bool is_pdd_coef1_{false};
+  bool is_pdd_coef2_{false};
+  bool is_leak_coef_{false};
+  Eigen::VectorXd pdd_poly_coef_1_;
```

Comment:
Vector of coefficients?

Reply:
Oh the bools are not coefficients. They are flags for me to check if coefficients are properly constructed. I guess I can call the construction function inside class constructor instead so I dont need these flags